test/discovery-store-model-test.js

```diff
@@ -346,16 +346,24 @@ describe('discovery-store-model', () => {
     it('converts a plain bib into an object with correctly grouped statements, including holdings', async () => {
       const bib = await platformApi.bibById('sierra-nypl', '12959619')
       const groupedStatements = await discoveryStoreModel.buildDiscoveryStoreBibs([bib])
-
       expect(groupedStatements).to.be.a('array')
       expect(groupedStatements[0]).to.be.a('object')
       expectStatement(groupedStatements[0]._statements, {
         predicate: 'dcterms:title',
         object_literal: 'AAHGS news : the bi-monthly newsletter of the Afro-American Historical and Genealogical Society, Inc.'
       })
-
+      const realItems = groupedStatements[0]._items.filter(i => !i.id.includes('i-h'))
+      const checkInCardItems = groupedStatements[0]._items.filter(i => i.id.includes('i-h'))
```

Comment:
This is fine. It relies on the convention for generating checkin card ids, which will be pretty static, but conceivably could change. So you could use the `rdfs:type` check from below to separate groups, e.g.:
```
const isCheckinCard = (item) => item._statements.some((s) => {
  return s.predicate === 'rdfs:type' &&
    s.object_id === 'nypl:CheckinCardItem'
})
const realItems = groupedStatmeents[0]._items.filter((g) => !isCheckinCard(g))
const checkinCardItems = groupedStatements[0]._items.filter(isCheckinCard)
```

Reply:
added